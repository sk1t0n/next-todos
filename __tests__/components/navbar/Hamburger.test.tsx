import { render } from '@testing-library/react';
import Hamburger from '../../../components/navbar/hamburger';

describe('Hamburger', () => {
  it('check the snapshot', () => {
    const { asFragment } = render(
      <Hamburger ref={null} onClick={(() => {})} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
