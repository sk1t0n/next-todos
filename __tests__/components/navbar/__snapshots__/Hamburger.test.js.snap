// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hamburger check the snapshot 1`] = `
<DocumentFragment>
  <div
    class="hamburger__Button-sc-5giryv-0 dKTYzt"
  >
    <div
      class="hamburger__Line1-sc-5giryv-1 epDPMR"
    />
    <div
      class="hamburger__Line2-sc-5giryv-2 shOHl"
    />
    <div
      class="hamburger__Line3-sc-5giryv-3 ijQAHE"
    />
  </div>
</DocumentFragment>
`;
